pulp_rpm/tests/functional/constants.py

```diff
@@ -55,6 +49,18 @@
 }
 """The metadata for one RPM package."""
 
+RPM_PACKAGE_NAME = '{}'.format(RPM_PACKAGE_DATA['name'])
+"""The name of one RPM package."""
+
+RPM_PACKAGE_FILENAME = '{}-{}{}-{}.{}.rpm'.format(
+    RPM_PACKAGE_DATA['name'],
+    RPM_PACKAGE_DATA['epoch'] + '!' if RPM_PACKAGE_DATA['epoch'] != '0' else '',
```

Comment:
I'm confused about the '!' character, I know that's used with Python epochs but it isn't used with RPM epochs to my knowledge.  I could be wrong but the ```gen_rpm_filename()``` function represents my knowledge of how it's supposed to be done according to the Fedora packaging guidelines. 

Maybe ```'' if RPM_PACKAGE_DATA['epoch'] else ':' + RPM_PACKAGE_DATA['epoch']```?

Reply:
I changed it using what we already have in place for Pulp 2 as example . I will evaluate what you mentioned. Thanks.

Reply:
Cool.

I used this to find RPMs with epoch set on my system

```rpm -qa --qf '%{E}:%{N}\n' | grep -v none```

and then if you try to install them, dnf will provide the filename

```Package python3-bind-32:9.11.4-10.P2.fc29.noarch is already installed.```

Reply:
I am reading about it to update the PR properly.

Reply:
For future reference: https://fedoraproject.org/wiki/Packaging:Versioning